heroic-core/src/main/java/com/spotify/heroic/ws/MandatoryClientIdErrorMessage.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2015 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.heroic.ws;
+
+import javax.ws.rs.core.Response;
+
+public class MandatoryClientIdErrorMessage extends ErrorMessage {
+    public MandatoryClientIdErrorMessage(final String message, final Response.Status status) {
```

Comment:
Would there be any case where the Response.Status where client id errors wouldn't be 400? If not, maybe we can just hard code 400 here  like
```
public MandatoryClientIdErrorMessage(final String message) {
    super(message, Status.BAD_REQUEST);
}
```

Reply:
good call